theatre_cli/config.py

```diff
@@ -1,6 +1,8 @@
 import json
-
-CONFIG_FILE = "config.json"
+import os
+#Changing the target path to the new folder that we created accodring to linux folder structure conventions
```

Comment:
@lexcsam Can you also give 2 lines space after `import` statements?
```suggestion
import os


#Changing the target path to the new folder that we created accodring to linux folder structure conventions
```